### ISSUE TITLE
Support chargepoint information feedback loop, requesting CPO (or MSP) to fix it

### ISSUE THREAD

**Issue by rutgerplantenga** [ocpi/ocpi#66]
Allow user (via MSP) to request CPO (and/or MSP) to review a chargepoint's status/information.

When chargepoint information is wrong, it is best fixed at the source. In most cases this means that the CPO needs to review and correct it. But there may also be cases where the MSP would need to correct it's interpretation or visualisation of the data. 

In any case, this issue is very common and frustrating and we have a chance to improve it.

Suggestion is to allow user (in MSP's app/web) to mark a chargepoint for review and mark an option to indicate what's wrong (without allowing user to enter text). 

Options could include:
1. Chargepoint is not functioning properly
2. Chargepoint cannot be accessed
3. Chargepoint status information is wrong
4. Chargepoint location information is wrong
5. Chargepoint connector/power details are wrong
6. Chargepoint pricing information is wrong
7. Chargepoint opening hours information is wrong

Then, the MSP should be able to mark the chargepoint/EVSE for review, including the reason.

**Comment by mbayings** [comment]
I agree with this Rutger, but I think this is something for next version and not for 2.0 right?

**Comment by simonschllng** [comment]
Good proposal! Many CPOs however are afraid that there could be too much feedback which they then have to cope with manually. Therefor it has to be well defined. We might find some inspiration in the feedback function of Google Maps. If some of the MSPs or CPOs have experiences with this topic they should share it.

**Comment by rutgerplantenga** [comment]
@mbayings agree, this cannot be for 2.0 (frozen) so for the next version. 

@simonschllng I fully agree. I have some experience with community content sourcing & feedback and we really need to avoid the overhead of moderation (and especially the communication about the suggested correction) and simplify/automate it as much as possible, also avoiding 'language' as a complicating factor.

But I think the way I proposed it will do exactly that. It would really just be that a chargepoint has flags/statuses indicating the things that need to be reviewed. Then, once the CPO (or MSP) checked/fixed it, they can remove the flag/reset the status. 

But I agree, if anyone has other ideas, please comment on this!

**Comment by cristinarc** [comment]
I think this idea is really interesting because it removes the necessity of making a call or sending an email to the CPO to inform about that. The MSP can still decide wether it needs to be sent to the CPO or not, meaning that, the MSP can have different ways of validating it is true or it isn't: user with reliable reviews, more than one user commented the same thing about the charge point, etc.

It is simple and it still solves the issue. So +1 from my side.
